src/components/UI/UIItems.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Button, Paper } from "@mui/material";
```

Comment:
is it component?

Reply:
it more looks like util

Reply:
I've understood, my failed. It's customization of material components. Thanks =)